fizzbuzz/fizzbuzz.c

```diff
@@ -10,7 +10,27 @@
 */
 int fizzbuzz(int n)
 {
-
+    int counter = 0;
+    for (int i = 1; i <= n; i++)
+    {
+        if (i % 3 == 0)
+        {
+            printf("Fizz\n");
+        }
+        else if (i % 5 == 0)
+        {
+            printf("Buzz\n");
+        }
+        else if (i % 3 == 0 && i % 5 == 0)
+        {
```

Comment:
I think you need to put this first. Otherwise nothing will ever trigger fizzbuzz - if you enter "15" it will immediately just print out fizz and just stop because you already met a condition. Does that make sense?